internal/common/common_test.go

```diff
@@ -1,4 +1,4 @@
-package common
+package common_test
```

Comment:
I think when possible it's best to change package name for testing, it has at least two major benefits:
* reduce dependencies: tests often need more dependencies
* only test public interface: it's good practice to only test public interfaces, so that we can change internal implementation as long as the public interface keeps the same behaviour

In this case it was a leftover because I moved away from common a function that we tested here (IIRC it was HasOperator) which would've caused circular dependencies. I could've reverted the package renaming but I have decided to leave the renamed package as IMO it only brings advantages. WDYT?

Reply:
I understand and agree with you, but it will probably be a bit confusing given the vast majority of tests are in the same package in this repo. If you don't need to change it I'd leave it in `common`.